manifests/apache.pp

```diff
@@ -1,100 +1,18 @@
 # Setups Apache for Katello development
 class katello_devel::apache {
 
-  include apache
-
-  $priority = '05'
-
-  $proxy_pass_https = [
-    {
-      'no_proxy_uris' => ['/pulp', '/streamer', '/pub'],
-      'path'          => '/',
-      'url'           => "http://localhost:${katello_devel::rails_port}/",
-      'params'        => {'retry' => '0'},
-    },
-    {
-      'path'          => '/',
-      'url'           => 'http://localhost:6006/',
-      'params'        => {'retry' => '0'},
-    },
-  ]
-
-  file { "${apache::confd_dir}/${priority}-foreman-ssl.d":
-    ensure  => 'directory',
-    owner   => 'root',
-    group   => 'root',
-    mode    => '0644',
-    purge   => true,
-    recurse => true,
-  }
-
-  apache::vhost { 'katello-ssl':
-    servername            => $certs::apache::hostname,
-    serveraliases         => $certs::apache::cname,
-    docroot               => '/var/www',
-    port                  => 443,
-    priority              => $priority,
-    options               => ['SymLinksIfOwnerMatch'],
-    ssl                   => true,
-    ssl_cert              => $certs::apache::apache_cert,
-    ssl_key               => $certs::apache::apache_key,
-    ssl_ca                => $certs::apache::ca_cert,
-    ssl_verify_client     => 'optional',
-    ssl_options           => '+StdEnvVars',
-    ssl_verify_depth      => '3',
-    custom_fragment       => file('katello/katello-apache-ssl.conf'),
-    additional_includes   => ["${apache::confd_dir}/${priority}-foreman-ssl.d/*.conf"],
-    use_optional_includes => true,
-    ssl_proxyengine       => true,
-    proxy_pass            => $proxy_pass_https,
-    proxy_preserve_host   => true,
-    request_headers       => ["set X_FORWARDED_PROTO 'https'"],
+  class { 'foreman::config::apache':
+    passenger     => false,
+    ssl           => true,
+    ssl_cert      => $certs::apache::apache_cert,
+    ssl_key       => $certs::apache::apache_key,
+    ssl_ca        => $certs::apache::ca_cert,
```

Comment:
If we want to mirror the production setup (which supports custom certs), should this also set the chain? I believe this should fix it.
```suggestion
    ssl_ca        => $certs::apache::ca_cert,
    ssl_chain     => $certs::apache::apache_ca_certs,
```